### PR TITLE
Internationalization retrofit

### PR DESCRIPTION
**Pull request by fjarrett**
**Goal:** Get the plugin translation-ready as quickly as possible.

* Retrofit existing template system to support standard WP i18n
* Wrap strings in PHP files with i18n functions
* Load textdomain on plugin bootstrap
* Create POT file

### CODE REVIEW

File: src/strings.php [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+__('E-mail:', 'sucuri-scanner');
+__('DNS Lookups', 'sucuri-scanner');
+__('Check the box if your website is behind a known firewall service, this guarantees that the IP address of your visitors will be detected correctly for the security logs. You can change this later from the settings.', 'sucuri-scanner');
+__('Enable DNS Lookups On Startup', 'sucuri-scanner');
+__('I agree to the <a target="_blank" href="https://sucuri.net/terms">Terms of Service</a>.', 'sucuri-scanner');
+__('I have read and understand the <a target="_blank" href="https://sucuri.net/privacy">Privacy Policy</a>.', 'sucuri-scanner');
+__('Submit', 'sucuri-scanner');
+
+// settings-alerts-bruteforce.html.tpl
+__('Password Guessing Brute Force Attacks', 'sucuri-scanner');
```
Comment (yaelcampo) [inline src/strings.php]: Many of the strings in `strings.php` are not included in `sucuri-scanner.pot`, such as 'Password Guessing Brute Force Attacks'

File: src/template.lib.php [hunk 2]
```diff
@@ -81,6 +81,22 @@ private static function replacePseudoVars($content = '', $params = array())
...
+        global $locale;
+
+        if ( $locale !== 'en_US' ) {
+            require_once __DIR__ . '/strings.php';
+
+            preg_match_all('~{{(.+)}}~', $content, $matches);
+
+            if ( ! empty( $matches[1] ) ) {
+                foreach($matches[1] as $string) {
+                    $pattern = sprintf('~{{%s}}~', preg_quote($string));
```
Comment (yaelcampo) [inline src/template.lib.php]: It should be `preg_quote($string, '~')`

File: src/template.lib.php [hunk 3]
```diff
@@ -81,6 +81,22 @@ private static function replacePseudoVars($content = '', $params = array())
             }
         }
 
+        global $locale;
+
+        if ( $locale !== 'en_US' ) {
```
Comment (yaelcampo) [inline src/template.lib.php]: When `$locale == 'en_US'` we're not removing the `{{` `}}`

Comment (fjarrett) [reply src/template.lib.php]: Oh thanks! Fixed in https://github.com/Sucuri/sucuri-wordpress-plugin/pull/72/commits/ab7bec3b77251e800e2ea9f6cb9aceaa2aa1b50b

File: src/template.lib.php (hunk 2 above)
Comment (fjarrett) [reply src/template.lib.php]: Fixed in https://github.com/Sucuri/sucuri-wordpress-plugin/pull/72/commits/ab7bec3b77251e800e2ea9f6cb9aceaa2aa1b50b

File: src/strings.php (hunk 1 above)
Comment (fjarrett) [reply src/strings.php]: Needed a POT update, done in https://github.com/Sucuri/sucuri-wordpress-plugin/pull/72/commits/ab9d46deb4d416f4aa0565fd6fd8a51c4b15f1a5

**Comment by fjarrett** [comment]
@yaelcampo Thanks for the feedback on this! Addressed your comments.

File: src/template.lib.php [hunk 4]
```diff
@@ -81,6 +81,20 @@ private static function replacePseudoVars($content = '', $params = array())
             }
         }
 
+        global $locale;
+
+        require_once __DIR__ . '/strings.php';
```
Comment (yaelcampo) [inline src/template.lib.php]: Seems like this require_once is not necessary

Comment (fjarrett) [reply src/template.lib.php]: @yaelcampo oh good call, yeah no reason to load

**Pull request merged**